daemon/graphdriver/quota/projectquota.go

```diff
@@ -1,4 +1,4 @@
-// +build linux
+// +build linux,!no_quota_support
```

Comment:
I'd suggest `exclude_disk_quota`, as
 * we already have a few `exclude_...` tags (like `exclude_graphdriver_devmapper` etc)
 * `quota` is too generic
 * `support` is redundant

NB: too bad we don't have a place to document all docker build tags :-\

Reply:
updated